_posts/2015-03-10-updated-site.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title:  "Site Updates and Changes"
+date:   2015-03-10 12:21:00
+categories: updates
+author: polymer-team
+excerpt: We're updating the Polymer site a bit as 0.8 is nearing release.
+
+---
+
+As many have already noticed, over the last day or so we've been updating the [Polymer site](https://www.polymer-project.org) to prepare for the upcoming 0.8 release. Some old links have broken in the process - we apologize for any troubles!
+
+The 0.8 release is a big step towards a 1.0: we're rolling out changes to the site to create a place to show 0.8 documentation, to enable better support for versioning, and generally to create a more stable home that will be able to grow with the library.
+
+# What to expect with the site for 0.8
+
+0.8 will be a new baseline for feedback - it will be hot out of the oven, but able to stand on its own. Consider it an "alpha" release - rough around the edges but generally whole, with the express purpose of getting feedback to roll into 1.0.
+
+With this in mind, we're working to accompany the release with a baseline of documentation to help early-adopters get started - expect the documentation to grow significantly as the features bake. The site itself for the time being will continue to be built based on 0.5 - we want to get 0.8 out for wide feedback as soon as possible, so having some documentation ready and starting to create 0.8-compatible elements has taken a priority over updating the site's library itself.
```

Comment:
updating the site's library itself. -> over making the site work under the new release